EKF/ekf_helper.cpp

```diff
@@ -1479,6 +1479,7 @@ void Ekf::stopFlowFusion()
 	_flow_innov.setZero();
 	_flow_innov_var.setZero();
 	_optflow_test_ratio = 0.0f;
+	_flow_data_ready = false;
```

Comment:
This seems a bit problematic to me. We should use the control_status.flag.opt_control where needed.

Reply:
Yes, that's right. In fact, we already check for the status flag so this should not be needed